services/config_data_sources/json_project_data_source.rb

```diff
@@ -164,6 +166,7 @@ def create_project!(name: nil, repo_config: nil, enabled: nil, lane: nil, artifa
                                 project_name: name,
                                 lane: lane,
                                 artifact_provider: artifact_provider)
+      _ = GitRepo
```

Comment:
What does this do

Reply:
This forces the clone of  the project, so we can (for the moment) retrieve the Fastfile and its lanes.

Reply:
Ahhh let's make that more explicit... like make a method that specifically says `force_clone` or something